CoreAttributeEditorView/src/au/gov/asd/tac/constellation/views/attributeeditor/AttributeEditorTopComponent.java

```diff
@@ -248,7 +248,7 @@ public UndoRedo getUndoRedo() {
     }
 
     @Override
-    public void preferenceChange(PreferenceChangeEvent event) {
+    protected void handlePreferenceChange(final PreferenceChangeEvent event) {

```

Comment:
Again, this should be handled by the framework.

Reply:
Neat!